docker.md

```diff
@@ -35,30 +38,19 @@ We are not maintaining a central copy for now. Till then please main your own co
 DATABASE_NAME=<Name you are going to give your db>
 DATABASE_USER=root
 DATABASE_PASSWORD=<value of MYSQL_ROOT_PASSWORD>
-DATABASE_HOST=0.0.0.0(or host.docker.internal for windows)
+DATABASE_HOST=0.0.0.0(or host.docker.internal for windows or MacOs)
 DATABASE_PORT=3306
+REDIS_HOST=0.0.0.0(or host.docker.internal for windows or MacOs)
+REDIS_PORT=6379
 MYSQL_ROOT_PASSWORD=<This should be set>
+MYSQL_DATABASE=<Same as DATABASE_NAME>
 ```
 
 3. Bring your containers up  
 `docker-compose up -d`
-
    
-4. Now you can exec into the mysql container and create your database.
-```
-docker exec -it amos-ss2021-emba-service_auth-db_1 bash
-mysql -p
-```
-When prompted for a password enter the value in `MYSQL_ROOT_PASSWORD`  
-`Create database <value in DATABASE_NAME>`
-
-5. Restart your containers. This additional step is neccessary since there is no dependency between the containers. And django would try to connect to DB not there yet and fail.  
-`docker-compose restart emba`
-
-6. Exec into your emba-django container  
-`docker exec -it amos-ss2021-emba-service_emba_1 bash`
```

Comment:
Why is this deleted ?

Reply:
Not needed anymore. On first build database will come up automatically.  New Variable `MY_SQL_DATABASE` added in env does this.

Reply:
Ok nice, didnt know.